lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -164,43 +217,87 @@
 
 
 .object-type-filter-link {
-    padding: 10px 15px;
     font-size: 1.3em;
+    padding: 18px 15px;
 
     &:global(.active),
     &:hover {
-        text-decoration: none;
-        background-color: $color-bg-gray;
+        background-color: $color-bg-white;
+        border-bottom: 1px solid $color-text-black;
         color: $color-text-black;
-        border-bottom: 2px solid $color-text-black;
+        text-decoration: none;
     }
 }
 
 .sort-dropdown {
-    width: 170px;
-    margin-top: 10px;
+    background-color: $color-bg-gray-blue-light;
+
+    p {
+        margin: 0 0 0 2px;
+    }
+
+    div {
+        background-color: $color-bg-gray-blue-light;
+        border: 0 solid transparent;
+        color: $color-bg-blue-dark;
+        width: 170px;
+    }
+
+    > div {
+        display: flex;
+    }
+}
+
+.ember-power-select-status-icon {
+    right: 10px;
+    top: 12px;
 }
 
 .sidenav-toggle {
     float: left;
 }
 
 .no-results {
-    margin: 10px 20px;
     font-size: 1.2em;
+    margin: 10px 20px;
+}
+
+.left-panel-mobile {
+    padding: 1rem;
+    width: 75vw;
+
+    div:nth-of-type(2) {
+        margin-bottom: 1rem;
+    }
+
+    div[role='button'] {
+        width: 195px;
+    }
+}
+
+.ember-power-select-options {
+    max-height: 20em;
 }
 
 .left-panel {
-    padding-left: 12px;
-    width: 300px;
+    background-color: $color-bg-gray-blue-light;
+    margin-bottom: 0;
     min-width: 300px;
+    padding: 0 12px;
+    width: 300px;
+
+    h2 {
+        border-bottom: 1px solid #ddd;
+        font-weight: 300;
+        margin-bottom: 0;
+        padding-bottom: 0.5rem;
+    }
 }
 
 .search-count {
-    margin-top: 10px;
-    margin-bottom: 10px;
     font-size: 1.3em;
     font-weight: bold;
+    margin-top: 5px 0;
```

Comment:
`margin-top` can't have two values, can it? Should this be just be `margin`?

Reply:
Correct, this is now updated not just in-browser.